src/app/pages/subscriptions/confirm-subscription-wrapper.jsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import { string } from 'prop-types';
-
+import {
+  Box,
+} from '@material-ui/core';
 import ChplConfirmSubscription from './confirm-subscription';
-
 import AppWrapper from 'app-wrapper';
 
 function ChplConfirmSubscriptionWrapper({ hash }) {
   return (
     <AppWrapper>
+      <Box sx={{ backgroundColor: '#f9f9f9'}}>
```

Comment:
Nothing should go in these wrapper files that we want to keep. Eventually, all the wrapper files will be deleted, and if we just delete this one now this box and whatever it's doing will be lost. This should be in the "confirm-subscription" file somewhere